Makefile

```diff
@@ -0,0 +1,27 @@
+SHELL := /bin/bash
+
+.PHONY: help
+.DEFAULT_GOAL := help
+
+ENV_FILE := .env
+ifeq ($(filter $(MAKECMDGOALS),config clean),)
+	ifneq ($(strip $(wildcard $(ENV_FILE))),)
+		ifneq ($(MAKECMDGOALS),config)
+			include $(ENV_FILE)
+			export
+		endif
+	endif
+endif
+
+help: ## 💬 This help message :)
+	@grep -E '[a-zA-Z_-]+:.*?## .*$$' $(firstword $(MAKEFILE_LIST)) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-23s\033[0m %s\n", $$1, $$2}'
+
+ci: unittest build-frontend ## 🚀 Continuous Integration (called by Github Actions)
```

Comment:
Should we also run formatting and linting checks as part of this?

Reply:
This will fail the build if I add them here as another pr needs to fix the code first, but it's in a few days we will turn that on here too